src/main/java/com/google/sps/servlets/NewUserServlet.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
+
+
+@WebServlet("/newUser")
+public class NewUserServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    Entity userEntity = new Entity("user");
+
+    String name = request.getParameter("name");
+    String email = request.getParameter("email");
+    String username = request.getParameter("username");
+    String password = request.getParameter("password");
+    String contactNum = request.getParameter("contactNum");
+    String organization = request.getParameter("organization");
+
+    userEntity.setProperty("name", name);
+    userEntity.setProperty("username", username);
+    userEntity.setProperty("email", email);
+    userEntity.setProperty("password", password);
+    userEntity.setProperty("contactNumber", contactNum);
+    userEntity.setProperty("organization", organization);    
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(userEntity);
+
+    System.out.println("successfully added NEW USER into database");    
```

Comment:
I know this is for you to check functionality, but let's not include it when merging with master, or if you want, log it into the js console.

Reply:
I'll remove it.